src/main/java/nl/cherement/jak/controller/UserController.java

```diff
@@ -23,32 +20,25 @@
         return userService.findAllShortened();
     }
 
-    @GetMapping("/me")
-    public UserEntity findUser(Principal principal) {
```

Comment:
In Principal zit niet alle informatie van een UserEntity aleen het nodige voor verificatie

Reply:
Dat klopt. Maar we hebben deze methode nergens voor nodig toch? In wat voor situatie zouden we hem gebruiken?

Reply:
Op het moment als de gebruiker zijn informatie wilt inzien op zijn instellingen pagina(die nog niet bestaat). 
Maar je hebt gelijk de kans dat we hem echt nodig hebben is erg klein.

Reply:
Das waar. Maar dan kan hij opzich altijd weer worden toegevoegd

Reply:
Okey laten we het hier bij zitten.